google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1alpha2/BQTableSchemaToProtoDescriptorTest.java

```diff
@@ -201,7 +201,7 @@ public void testStructComplex() throws Exception {
             .build();
     final Table.TableFieldSchema TEST_TIME =
         Table.TableFieldSchema.newBuilder()
-            .setType(Table.TableFieldSchema.Type.TIME)
+            .setType(Table.TableFieldSchema.Type.INT64)
```

Comment:
Why do we have this change?

Reply:
I made this change to get the test to pass, it's kind of cheesy but I figured the alpha test and such a small change would be okay. It was needed because of the changes made to the proto file that the fields are checked against.

Reply:
I see. That's OK.